src/Controller/AnonymizationController.php

```diff
@@ -25,27 +26,27 @@ class AnonymizationController extends AbstractController
 
     private EventDispatcherInterface $eventDispatcher;
 
-    private ParameterBagInterface $parameterBag;
+    private TranslatorInterface $translator;
 
     public function __construct(
         EntityManagerInterface $entityManager,
         CustomerRepositoryInterface $customerRepository,
         AnonymizerInterface $anonymizer,
         EventDispatcherInterface $eventDispatcher,
-        ParameterBagInterface $parameterBag
+        TranslatorInterface $translator
     ) {
         $this->entityManager = $entityManager;
         $this->customerRepository = $customerRepository;
         $this->anonymizer = $anonymizer;
         $this->eventDispatcher = $eventDispatcher;
-        $this->parameterBag = $parameterBag;
+        $this->translator = $translator;
     }
 
-    public function __invoke(string $id): Response
+    public function __invoke(Request $request, string $id): Response
     {
         $customer = $this->customerRepository->find($id);
         if (!$customer instanceof CustomerInterface) {
-            $this->parameterBag->set('error', 'sylius.ui.admin.sylius_gdpr.customer.not_found');
+            $request->getSession()->getFlashBag()->add('error', $this->translator->trans('sylius.ui.admin.sylius_gdpr.customer.not_found'));
```

Comment:
you must put the translation inside flashes.en.yml

Reply:
Yes, it's fixed.